Tests/BugsnagConfigurationTests.m

```diff
@@ -52,4 +52,16 @@ - (void)testNotifyReleaseStagesExcludedSkipsSending {
     XCTAssertFalse([config shouldSendReports]);
 }
 
+- (void)testLaunchThreshold {
+    BugsnagConfiguration *config = [BugsnagConfiguration new];
+    XCTAssertEqual(10000, config.launchCrashThresholdMs);
+    
+    config.launchCrashThresholdMs = 0;
+    XCTAssertEqual(0, config.launchCrashThresholdMs);
+    
+    int expected = 1500;
```

Comment:
`expected` here is `int` type yet we are comparing it to `launchCrashThresholdMs` which is `NSUInteger`.